src/routes/sessions/[id]/+page.svelte

```diff
@@ -169,12 +170,27 @@
 <div class="session">
 	<Header>
 		<p data-testid="session-id" class="text-sm font-bold leading-none">
-			Session <Button size="link" variant="link" href={`/${session.id}`}>#{session.id}</Button>
+			Session <Button size="link" variant="link" href={`/sessions/${session.id}`}
+				>#{session.id}</Button
+			>
 		</p>
-		<p data-testid="model-name" class="text-sm text-muted">
-			{isNewSession ? 'New session' : session.model}
-		</p>
-
+		<div class="grid grid-cols-[auto,auto] gap-x-1">
+			<!-- {#if isNewSession}
+				<p data-testid="new-session-text" class="text-sm text-muted">New session</p>
+			{:else}
+				{#if session.updatedAt}
+					<Badge variant="default" capitalize={false}>
+						{formatShortDistanceToNow(new Date(session.updatedAt), { addSuffix: true })}
+					</Badge>
+				{/if}
+				<Badge variant="default" capitalize={false}>
+					<p data-testid="model-name">{session.model}</p>
+				</Badge>
+			{/if} -->
+			<p data-testid="model-name" class="text-sm text-muted">
+				{isNewSession ? 'New session' : session.model}
+			</p>
```

Comment:
This is now only showing the model but not the timestamp.

I think ideally the `<Header>` metadata should be identical to the one on the sidebar so we can probably add a new function `export function formatSessionMetadata(session: Session)` in `src/lib/sessions.ts` that has all of this logic:

```svelte
{@const subtitles = [
	session.model,
	...(knowledgeName ? [knowledgeName] : []),
	...(session.updatedAt
		? [formatDistanceToNow(new Date(session.updatedAt), { addSuffix: true })]
		: [])
]}
```

And since we don't have to use the funky Svelte syntax we can do something a bit more readable like:

```typescript
export function formatSessionMetadata(session: Session)
    const subtitles = [session.model];
    if (knowledgeName) subtitles.push(knowledgeName);
    if (session.updatedAt) subtitles.push(formatTimestamp(session.updatedAt));
    return subtitles.join(' • ')
)
```